INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 133
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:60 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:60 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:60 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:60 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 15396 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:51821
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element/f.2E20BFF4720160D449D45E0092373556.d.EDE7500288E16430A4EB4E5E5D4537C9.e.2/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element/f.2E20BFF4720160D449D45E0092373556.d.EDE7500288E16430A4EB4E5E5D4537C9.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element/f.2E20BFF4720160D449D45E0092373556.d.EDE7500288E16430A4EB4E5E5D4537C9.e.6/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 860171bc1d1cbcc0c03e885e9c1446f358f2e35f1e0f2bac33958c25227f2986
DEBUG    LoginPage:login_page.py:32 login crumb: 860171bc1d1cbcc0c03e885e9c1446f358f2e35f1e0f2bac33958c25227f2986
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/title HTTP/1.1" 200 0
DEBUG    Header:ui_element.py:56 Click on locator ('xpath', '//a[@href="/logout"]')
DEBUG    Header:ui_element.py:28 waiting for element_to_be_clickable(('xpath', '//a[@href="/logout"]'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/element/f.15916D8580B37FA0299F4F90773AD8FD.d.B01194273CC0912217833CF5819A6F92.e.52/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element/f.15916D8580B37FA0299F4F90773AD8FD.d.B01194273CC0912217833CF5819A6F92.e.52/click HTTP/1.1" 200 0
DEBUG    LoginPage:ui_element.py:28 waiting for text_to_be_present_in_element(('xpath', '//main//h1'), 'Sign in to Jenkins')
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/element/f.15916D8580B37FA0299F4F90773AD8FD.d.03ABB0A01AF9F0D10F5D18C3FCD78D56.e.11/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "POST /session/2e0b48225693b0b1bc31aaabba2d2b52/refresh HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "GET /session/2e0b48225693b0b1bc31aaabba2d2b52/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:51821 "DELETE /session/2e0b48225693b0b1bc31aaabba2d2b52 HTTP/1.1" 200 0